hack/update-all.sh

```diff
@@ -62,9 +62,9 @@ BASH_TARGETS="
 	update-generated-swagger-docs
 	update-swagger-spec
 	update-openapi-spec
-	update-api-reference-docs
-	update-staging-godeps
-	update-bazel"
+	update-api-reference-docs"
+#	update-staging-godeps
+#	update-bazel"
```

Comment:
Why are these commented out?

Reply:
Sorry. I will remove that. update-bazel hasn't worked for me since I switched to go 1.9. I just wanted to be able to run the script locally.